Cargo.toml

```
@@ -115,13 +115,16 @@ trin-storage = { path = "crates/storage" }
 trin-utils = { path = "crates/utils" }
 trin-validation = { path = "crates/validation" }
 
-# TODO: When we build for a windows target on an ubuntu runner, crunchy tries to
-# get the wrong path, update this when the workflow has been updated
-#
-# See: https://github.com/eira-fransham/crunchy/issues/13
 crunchy = "=0.2.2"
 
-# todo: remove this when our other dependencies update to getrandom 0.3 as it is a breaking change
 [patch.crates-io]
+# TODO: remove this when our other dependencies update to getrandom 0.3 as it is a breaking change
 tempfile = { git = "https://github.com/Stebalien/tempfile", tag = "v3.15.0" }
+# TODO: remove this when our other dependencies update to getrandom 0.3 as it is a breaking change
 uuid = { git = "https://github.com/uuid-rs/uuid", tag = "1.12.1" }
+
+# TODO: When we build for a windows target on an ubuntu runner, crunchy tries to
+# get the wrong path, update this when the workflow has been updated
+#
+# See: https://github.com/eira-fransham/crunchy/issues/13
+crunchy = { git = "https://github.com/eira-fransham/crunchy", ref = "1bf90cf2d0a8cfcb2c5592275a23ab028dff6468" }
```

```suggestion
crunchy = { git = "https://github.com/eira-fransham/crunchy", rev = "1bf90cf2d0a8cfcb2c5592275a23ab028dff6468" }
```

this must be rev instead of ref